### DISCUSSION TITLE
Help with Cyrillic

### DISCUSSION THREAD

**Discussion by BSK-YouTube** [MAFINS/MenyooSP#496] [Q&A]
Hello, I really like your trainer, perhaps the BEST trainer I've ever seen, I want to make a translation of the language for myself - but I can't display Cyrillic, how to implement it?
Currently showing squares instead of letters.

**Comment by BSK-YouTube** [comment]
Guys - I figured it out, the **topic is not relevant**.

**Comment by MAFINS** [comment]
Did you ever figure it out?

**Reply by BSK-YouTube** [reply]
I did it, thank you for all your hard work.
